starboard/android/shared/media_decoder.cc

```diff
@@ -102,7 +102,7 @@ MediaDecoder::MediaDecoder(Host* host,
   if (audio_stream_info.codec != kSbMediaAudioCodecOpus &&
       !audio_stream_info.audio_specific_config.empty()) {
     pending_tasks_.push_back(Event(audio_stream_info.audio_specific_config));
-    number_of_pending_tasks_.increment();
+    ++number_of_pending_tasks_;
```

Comment:
The increment() function does a post-increment, just like fetch_add(1). Using pre-increment here is fine because we don’t use the returned value. Plus, it’s better because pre-increment is faster than post-increment, which keeps a copy of the old value.